test/html-to-vdom/index.js

```diff
@@ -183,6 +183,33 @@ describe('htmlparser-to-vdom', function () {
         });
     });
 
+    describe('when converting a tag with a custom attributes', function () {
+        it('parses an item with a custom attribute correctly', function () {
+
+            var html = '<div e-custom="testing"></div>';
+
+            var converted = convertHTML(html);
+            converted.properties.attributes['e-custom'].should.eql('testing');
+        });
+
+        it('parses an item with a custom attribute without a value correctly', function () {
+
+            var html = '<div e-custom></div>';
+
+            var converted = convertHTML(html);
+            should.exist(converted.properties.attributes['e-custom']);
+        });
+    });
+
+    describe('when converting a tag with a camelCase valid property', function () {
+        it('parses correctly when given an all lowercase string', function () {
+            var html = '<div contenteditable></div>';
+
+            var converted = convertHTML(html);
+            should.exist(converted.properties.contenteditable);
```

Comment:
Shouldn't it be `converted.properties.contentEditable` according to [the specification](https://html.spec.whatwg.org/multipage/interaction.html#contenteditable)?